tests/e2e/specs/stories-editor/video-poster-image-extraction.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, getAllBlocks, selectBlockByClientId } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience } from '../../utils';
```

Comment:
Maybe we could have just `toggleExperience` instead? E.g. like `toggleScreenOption` in Gutenberg.

Then we could do just `if ( isChecked !== shouldBeChecked ) {` check to see if click or not.

Reply:
I see that you did this in #2824.

For me it felt more explicit to have two separate methods, like `activatePlugin` and `deactivatePlugin`.

Don't have a strong opinion about it though. 🤷‍♂

Reply:
For the plugin (de)activation two separate methods seem to make more sense to me as well.

Anyway -- we can also keep it separately, I also don't have a strong opinion about it, was mainly following how it was in Gutenberg for the settings.